marklogic-data-hub-central/ui/src/pages/Run.tsx

```diff
@@ -281,7 +283,8 @@ const Run = (props) => {
 
     function showFailed(stepName, stepType, errors) {
         Modal.error({
-            title: <div id="error-title"><p style={{fontWeight: 400}}>{formatStepType(stepType)} step <strong>{stepName}</strong> failed</p></div>,
+            title: <div id="error-title"><p style={{fontWeight: 400}}>The {stepType.toLowerCase()} step <strong>{stepName}</strong> failed</p></div>,
+            icon: <Icon type="exclamation-circle" theme="filled"/>, 
```

Comment:
e2e test locator needs update to reflect this change.
<img width="1089" alt="image" src="https://user-images.githubusercontent.com/16124406/90837845-074dbc80-e308-11ea-90c4-c146a257ebb6.png">

Reply:
@bsrikan thanks, I'm on it.